### PR TITLE
chore: Create 5.1.3.md

### PR DESCRIPTION
**Pull request by michael-wisely-gravwell**
This PR addresses no issue.

This PR proposes adding a changelog for v5.1.3

### CODE REVIEW

File: changelog/5.1.3.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+* Fixed an issue where live update interval form value is ignored in the query studio.
+* Fixed an issue where macro edit page save button is incorrectly disabled.
+* Fixed an issue where table would lock browser in loop.
+* Fixed an issue with suggestion selection on label input.
+* Fixed an issue where details pane total count was wrong after changing ax.
+* Fixed an issue where some ingester configs would be labelled as 'listener' configs.
+* Fixed an issue where flows could not be duplicated by non-admins if the flow was global.
+* Prevented context submenu from being cutoff in Query Studio view.
+* Updated username value in app.comp when user is nil.
```
Comment (john-floren-gravwell) [inline changelog/5.1.3.md:17]: what is app.comp?

File: changelog/5.1.3.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* Re-styled extra row data (EVs, tag, etc.) within details pane.
+* Added tooltip to show full value for long stats labels.
+* Fixed an issue where macro saved but did not return to the `/macros` list.
+* Fixed an issue where a Search Terminated error could be displayed twice.
+* Fixed an issue where Flow debug rows only partially highlight.
+* Fixed an issue where kit archive rebuild was not populating cover / banner / icon images.
+* Fixed an issue where duplicated tab would close search on source tab.
+* Fixed an issue where disk list form filters are reset.
+* Fixed an issue where the dashboard settings doesn't show the PermissionsStatus component for Actionables.
+* Fixed an issue where couldn't save toggles in dashboard settings.
```
Comment (john-floren-gravwell) [inline changelog/5.1.3.md:29]: grammar... "user couldn't save toggles", maybe?

File: changelog/5.1.3.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* Fixed an issue where live update interval form value is ignored in the query studio.
+* Fixed an issue where macro edit page save button is incorrectly disabled.
+* Fixed an issue where table would lock browser in loop.
+* Fixed an issue with suggestion selection on label input.
+* Fixed an issue where details pane total count was wrong after changing ax.
+* Fixed an issue where some ingester configs would be labelled as 'listener' configs.
+* Fixed an issue where flows could not be duplicated by non-admins if the flow was global.
+* Prevented context submenu from being cutoff in Query Studio view.
+* Updated username value in app.comp when user is nil.
+* Allowed users to change dashboard default variables before launching.
```
Comment (john-floren-gravwell) [inline changelog/5.1.3.md:18]: is this perhaps better in New Additions?

File: changelog/5.1.3.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* Made granularity disclaimer conditional for charts.
+* Re-styled extra row data (EVs, tag, etc.) within details pane.
+* Added tooltip to show full value for long stats labels.
```
Comment (john-floren-gravwell) [inline changelog/5.1.3.md:21]: New Additions?

File: changelog/5.1.3.md (hunk 1 above)
Comment (michael-wisely-gravwell) [reply changelog/5.1.3.md]: i rewrote that line to describe the bug that was fixed rather than the fix itself.

**Review by kris-watts-gravwell — APPROVED** [review]
LGTM

**Pull request merged**